app/assets/javascripts/admin/addon/components/admin-flag-item.gjs

```diff
@@ -47,15 +49,18 @@ export default class AdminFlagItem extends Component {
   @action
   toggleFlagEnabled(flag) {
     this.enabled = !this.enabled;
+    this.isSaving = true;
 
     return ajax(`/admin/config/flags/${flag.id}/toggle`, {
       type: "PUT",
     })
       .then(() => {
         this.args.flag.enabled = this.enabled;
+        this.isSaving = false;
       })
       .catch((error) => {
         this.enabled = !this.enabled;
+        this.isSaving = false;
```

Comment:
I think we can just do `finally(() => this.isSaving = false)` to avoid needing two lines here.